annotation-processor/src/main/resources/org/hibernate/validator/ap/ValidationProcessorMessages.properties

```diff
@@ -40,3 +40,6 @@ INVALID_DIGITS_ANNOTATION_PARAMETERS=Invalid annotation parameters. Fraction and
 INVALID_DECIMAL_MIN_MAX_ANNOTATION_PARAMETERS=Invalid annotation parameters. Value should be a valid BigDecimal string representation.
 INVALID_GROUPS_VALUE_ANNOTATION_PARAMETERS=Invalid annotation parameters. Groups should be represented by interfaces.
 INVALID_MESSAGE_VALUE_ANNOTATION_PARAMETERS=Message seems to be malformed. Did you miss a '{}' in the message key?
+INVALID_GROUP_SEQUENCE_VALUE_NOT_INTERFACES_ANNOTATION_PARAMETERS=Invalid annotation parameters. Value should be represented by interfaces.
+INVALID_GROUP_SEQUENCE_VALUE_CYCLIC_DEFINITION_ANNOTATION_PARAMETERS=Invalid annotation parameters. The defined group sequence should be expandable (no cyclic definition).
+INVALID_GROUP_SEQUENCE_VALUE_HOSTING_BEAN_ANNOTATION_PARAMETERS=Invalid annotation parameters. The class list should contain the hosting bean class (for default group sequence re-definition).
```

Comment:
The part "(for default group sequence re-definition)" could be omitted, it should be clear from the context of the error message.